static/js/lightbox_panzoom.js

```diff
@@ -0,0 +1,63 @@
+import Panzoom from '@panzoom/panzoom'
+
+import * as overlays from "./overlays";
+
+export function PanZoomControl(container) {
+    this.container = container
+    this.panzoom = Panzoom(this.container, {disablePan: true, disableZoom: true, cursor: "auto"})
+
+    this.reset = function(){
+        this.panzoom.reset()
+    }
+
+    this.disablePanZoom = function(){
+        this.container.removeEventListener("wheel", this.panzoom.zoomWithWheel)
+        this.panzoom.setOptions({disableZoom: true, disablePan: true, cursor: "auto"})
+        this.reset()
+    }
+
+    this.enablePanZoom = function(){
+        this.panzoom.setOptions({disableZoom: false, disablePan: false, cursor: "move"})
+        this.container.addEventListener("wheel", this.panzoom.zoomWithWheel)
+    }
+
+    this.zoomIn = function(){
+        this.panzoom.zoomIn()
+    }
+
+    this.zoomOut = function(){
+        this.panzoom.zoomOut()
+    }
+
+    // keybindings for zoom control
+    // Original comment from lightbox_canvas.js:
```

Comment:
I'm wondering if this is worth looking in to addressing here? The dependency on overlays and adding an eventListener here on keydown feels bad as it's pretty hidden away.